### PR TITLE
3614 discovery mock up of medical device tool

### PR DESCRIPTION
**Pull request by maharielrosario**
# Draft PR 

## Description
My team and I are creating an API to interact with the Denver Logistics Center, in order to send and receive data about hearing aid batteries, prosthetic socks, and potentially more products that can be ordered online. We are looking to implement Redux into our app and have already created Redux actions, types, a reducer, and a store to handle our state management and are looking for the VA's standard on how to best set up Redux.

## URL

http://localhost:3001/disability-benefits/2346/introduction

## Testing done
We have tested our endpoint using mock data and confirmed that it does send back data.

## Screenshots

screenshot-localhost_3001-2020 01

## Blockers

We need the 3820-Vets-API-Endpoint-Controller-Route branch merged as well

## Acceptance criteria
- As a front end developer, I want to fetch and update data from the DLC API and have this process of retrieving/updating data done using the VA's standard of Redux.

## Definition of done
- [ ] I am able to have DLC API data loaded into components
- [ ] If the GET or PUT request is successful, the data is stored in action.payload
- [ ] If the GET or PUT request has failed, the error message is stored in action.payload

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12849,7 +12849,7 @@ smart-buffer@4.0.2:
   resolved "https://registry.yarnpkg.com/smart-buffer/-/smart-buffer-4.0.2.tgz#5207858c3815cc69110703c6b94e46c15634395d"
   integrity sha512-JDhEpTKzXusOqXZ0BUIdH+CjFdO/CR3tLlf5CN34IypI+xMmXW1uB16OOY8z3cICbJlDAVJzNbwBhNO0wt9OAw==
 
-smart-buffer@^1.0.13:
+smart-buffer@^1.0.13
```
Comment (ncksllvn) [inline yarn.lock]: Looks like an accident

Comment (mr0sari0) [reply yarn.lock]: @ncksllvn thanks, I updated that in commit 392db1b

**Comment by rianfowler** [comment]
Is this still in progress? could we close this PR and move this branch to a draft PR if it is not ready?

**Comment by mr0sari0** [comment]
Added screenshot and URL

**Comment by mr0sari0** [comment]
> Is this still in progress? could we close this PR and move this branch to a draft PR if it is not ready?

Sorry, I didn't assign the right devs.

**Comment by mojaray2k** [comment]
@rianfowler How do we change this to a draft PR?

**Comment by mr0sari0** [comment]
Just updating this as a draft PR.

**Comment by kelsonic** [comment]
GitHub doesn't have a way to go from an open PR to a draft PR 🙁So you would have to close this PR and then reopen a subsequent one in draft mode

**Comment by mr0sari0** [comment]
Closing this PR and opening a new draft PR as there is no way to convert this existing PR.

**Comment by mr0sari0** [comment]
New draft PR is located [here](https://github.com/department-of-veterans-affairs/vets-website/pull/11506)

**Pull request closed without merging**